pom.xml

```diff
@@ -121,6 +121,7 @@
                  <failsOnError>true</failsOnError>
                  <linkXRef>false</linkXRef>
                  <useFile/>
+                 <includeTestSourceDirectory>true</includeTestSourceDirectory>
```

Comment:
@stalep 
I don't know why we have pom.xml yet, since we work always with gradle.